lib/commands/sync.js

```diff
@@ -49,8 +49,19 @@ let HELPTEXT = `
                                               2 = Errors Only
                                               3 = No logs
 
+
+    ██████╗  █████╗ ████████╗██╗  ██╗███████╗██████╗ 
+    ██╔════╝ ██╔══██╗╚══██╔══╝██║  ██║██╔════╝██╔══██╗
+    ██║  ███╗███████║   ██║   ███████║█████╗  ██████╔╝
+    ██║   ██║██╔══██║   ██║   ██╔══██║██╔══╝  ██╔══██╗
+    ╚██████╔╝██║  ██║   ██║   ██║  ██║███████╗██║  ██║
+      ╚═════╝ ╚═╝  ╚═╝   ╚═╝   ╚═╝  ╚═╝╚══════╝╚═╝  ╚═╝
+                                                                                                
```

Comment:
![rainbow-eyes](https://user-images.githubusercontent.com/779325/58601064-2d93c200-8255-11e9-9a96-4af0bca493a9.gif)

Also we might want to talk about making this repo private

Reply:
_THE WORLD MUST KNOW_